src/test/java/com/wikia/webdriver/testcases/mercurytests/curatedcontenttests/EditorTests.java

```diff
@@ -0,0 +1,186 @@
+public class EditorTests extends NewTestTemplate {
+
+  @BeforeMethod(alwaysRun = true)
+  public void beforeMethod() {
+    wikiURL = urlBuilder.getUrlForWiki(MercuryWikis.MERCURY_EMPTY_CC_EDITOR);
+
+    new CuratedContent().clear();
+
+    // This login is temporary solution, use @Execute after QAART-669 is done
+    new LoginPage(driver).get().logUserIn(Configuration.getCredentials().userNameStaff2,
+                                          Configuration.getCredentials().passwordStaff2);
+  }
+
+  public static final String ITEM_DISPLAY_NAME = "Templates";
+  public static final String ITEM_PAGE_NAME = "Category:Templates";
+  public static final String SECTION_DISPLAY_NAME = "New Section";
+  public static final String SEARCH_IMAGE_QUERY = "U";
+
+  @Test(groups = "MercuryCuratedEditorTest_001")
+  @Execute(onWikia = "mercuryemptycceditor")
+  public void MercuryCuratedEditorTest_001_addAndSaveItemToFeaturedContent() {
+    CuratedMainPagePageObject curatedMainPagePageObject = new CuratedMainPagePageObject(driver);
+    EditorHomePageObject editorHomePageObject = new EditorHomePageObject(driver);
+
+    Boolean result = !curatedMainPagePageObject.isFeaturedContentVisible();
+    PageObjectLogging.log(
+        "Featured Content",
+        MercuryMessages.INVISIBLE_MSG,
+        MercuryMessages.VISIBLE_MSG,
+        result
+    );
+
+    curatedMainPagePageObject.navigateToUrlWithPath(wikiURL, MercuryPaths.ROOT_MAIN_EDIT);
+    ItemFormPageObject itemFormPageObject = editorHomePageObject.clickAddFeaturedContent();
+    itemFormPageObject.typeDisplayName(ITEM_DISPLAY_NAME);
+    itemFormPageObject.typePageName(ITEM_PAGE_NAME);
+
+    UploadImageModalComponentObject upload = itemFormPageObject.clickOnImage();
+    SearchForImagePageObject search = upload.clickSearchForImageButton();
+    search.type(SEARCH_IMAGE_QUERY);
+    CroppingToolPageObject croppingTool = search.clickOnImage(0);
+    croppingTool.clickDoneButton();
+
+    itemFormPageObject.waitForDeleteButtonToBeVisible();
+    itemFormPageObject.clickDone();
+    editorHomePageObject.waitForAddCategoryButtonToBeVisible();
+    editorHomePageObject.publish();
+
+    result = curatedMainPagePageObject.isFeaturedContentVisible();
+    PageObjectLogging.log(
+        "Featured Content",
+        MercuryMessages.VISIBLE_MSG,
+        MercuryMessages.INVISIBLE_MSG,
+        result
+    );
+  }
+
+  @Test(groups = "MercuryCuratedEditorTest_002")
+  @Execute(onWikia = "mercuryemptycceditor")
+  public void MercuryCuratedEditorTest_002_addAndSaveSection() {
+    CuratedMainPagePageObject curatedMainPagePageObject = new CuratedMainPagePageObject(driver);
+    CuratedContentPageObject curatedContentPageObject = new CuratedContentPageObject(driver);
+    EditorHomePageObject home = new EditorHomePageObject(driver);
+
+    Boolean result = !curatedMainPagePageObject.isCuratedContentVisible();
+    PageObjectLogging.log(
+        "Curated Content",
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Curated Content" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)